extension/packages/nextjs/app/dashboard/_components/CollateralGraph.tsx

```diff
@@ -102,7 +102,7 @@ const CollateralGraph = () => {
     const collateralAdded: bigint = event?.eventName === "CollateralAdded" ? event?.args.amount || 0n : 0n;
     const collateralWithdrawn: bigint = event?.eventName === "CollateralWithdrawn" ? event?.args.amount || 0n : 0n;
     const price: bigint =
-      "price" in event?.args ? event?.args.price : getPriceFromEvent(event?.blockNumber, priceEvents);
+      event?.args && event?.args.price ? event?.args.price : getPriceFromEvent(event?.blockNumber, priceEvents);
```

Comment:
nit, could be omitted: `event?.args?.price ?? getPriceFromEvent(event?.blockNumber, priceEvents);` should also work